dvc/command/diff.py

```diff
@@ -157,14 +135,15 @@ def add_parser(subparsers, parent_parser):
         ),
     )
     diff_parser.add_argument(
-        "a_ref", help="Git reference from which diff calculates"
+        "--json",
```

Comment:
How about `--show-json`, same as in `dvc metrics diff`?

Reply:
It is implicit that you are going to _show_ an output, the option is a format modifier, that's why I only named it `--json`.

However, I'm OK with changing it to `--show-json`.

What do you think, @efiop ?

Reply:
@mroutis In `dvc metrics diff` my logic was that I didn't want it to be confused with `-t json` and stuff like that. `--show-json` is more explicit, and, well, let's keep it consistent between the commands. So I would use `--show-json` for now.

Reply:
All righty)